### PR TITLE
cocoapods 1.8.1

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by akshayakrsh** [comment]
Possible to prioritize merging this?

**Comment by SMillerDev** [comment]
No it isn't, we'll have to wait until CI says it's okay. Which could take a while since all nodes were/are offline

File: Formula/cocoapods.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Cocoapods < Formula
   desc "Dependency manager for Cocoa projects"
   homepage "https://cocoapods.org/"
-  url "https://github.com/CocoaPods/CocoaPods/archive/1.7.5.tar.gz"
-  sha256 "508e5f7a7566b3d05ec4e27417dc0a60bedc8e72618b31cb56713034e71337b9"
+  url "https://github.com/CocoaPods/CocoaPods/archive/1.8.0.tar.gz"
+  sha256 "fb7031153dc464affccd83c9556d808a245280855154a7a5a4c234ffd33192bc"
```
Comment (kylef) [inline Formula/cocoapods.rb]:
```suggestion
  sha256 "bbca0489a41896253c579627772041fb7e725f642c39de27b4cc4dbc32d0c1ea"
```

File: Formula/cocoapods.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Cocoapods < Formula
   desc "Dependency manager for Cocoa projects"
   homepage "https://cocoapods.org/"
-  url "https://github.com/CocoaPods/CocoaPods/archive/1.7.5.tar.gz"
-  sha256 "508e5f7a7566b3d05ec4e27417dc0a60bedc8e72618b31cb56713034e71337b9"
+  url "https://github.com/CocoaPods/CocoaPods/archive/1.8.0.tar.gz"
```
Comment (kylef) [inline Formula/cocoapods.rb]:
```suggestion
  url "https://github.com/CocoaPods/CocoaPods/archive/1.8.1.tar.gz"
```

**Review by kylef — COMMENTED** [review]
There's a 1.8.1 release of CocoaPods now so perhaps this can be updated directly to that? Unless you would like to do it separately afterwards.

**Comment by SMillerDev** [comment]
I'll update the review. Thanks @kylef

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Pull request closed without merging**